### PR TITLE
Update SentinelOne-V2.py

### PR DESCRIPTION
**Pull request by pierremahot**
add limit to list agent by defaut it's 10

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
add limit to list agent by defaut it's 10

## Minimum version of Cortex XSOAR
- [x] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

**Review by Bargenish — CHANGES_REQUESTED** [review]
@pierremahot Thank you for your contribution.
Please add the `limit` argument to the command's arguments in the integration's yml file.
Also, add release notes and bump the pack version. Read [this](https://xsoar.pan.dev/docs/documentation/release-notes) for more information.

**Review by Bargenish — CHANGES_REQUESTED** [review]
@pierremahot Great, let's just revert the changes that were made to the **RubrikPolaris** Integration.

File: Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py [hunk 1]
```diff
@@ -958,6 +958,7 @@ def list_agents_command(client: Client, args: dict) -> CommandResults:
         scan_status=args.get('scan_status'),
         os_type=args.get('os_type'),
         created_at=args.get('created_at'),
+        limit=args.get('limit'),
```
Comment (Bargenish) [inline Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py:961]: Does the API recieve an int as the limit or a string?

Comment (pierremahot) [reply Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py]: yes I have change the code to cast to int

**Review by Bargenish — APPROVED** [review]

**Pull request merged**